### ISSUE TITLE
How to use tigris data on  cloudflare?

### ISSUE THREAD

**Issue by siddiquiaffan** [tigrisdata/tigris#1112]
As tigris is serverless noSql database, how to use it on serveless platforms like Cloudflare?

**Comment by ovaistariq** [comment]
@siddiquiaffan Tigris can be used with Cloudflare workers. Is that where you are thinking of using it?

**Comment by siddiquiaffan** [comment]
> @siddiquiaffan Tigris can be used with Cloudflare workers. Is that where you are thinking of using it?

Yes, I'm willing to use it on cloudflare workers. Is there any docs or example for it?

**Comment by ovaistariq** [comment]
@siddiquiaffan, we do not have documentation for it. But you should be able to use our [TypeScript](https://www.tigrisdata.com/docs/sdkstools/typescript/) client within the workers. I will happily work with you on any issues you face using Tigris in Cloudflare workers.

**Comment by siddiquiaffan** [comment]
Thanks will check it out!

**Comment by ovaistariq** [comment]
Great. Let me know if you run into any issues.
